### PR TITLE
Add embeddable url section

### PR DESCRIPTION
**Pull request by elv-zenia**
Load embed url and display at the top of the sidebar. This will update based on any offering selected from Advanced Controls.
8_15_2023_3-53-49 PM

### CODE REVIEW

File: src/stores/VideoStore.js [hunk 1]
```diff
@@ -310,6 +314,39 @@ class VideoStore {
...
+      embedUrl.searchParams.set("vid", versionHash);
+    } else if(objectId) {
+      embedUrl.searchParams.set("oid", objectId);
+    }
+
+    if(["owner", "editable", "viewable"].includes(permission)) {
+      const token = yield this.rootStore.client.CreateSignedToken({
+        objectId,
+        versionHash,
+        duration: 100 * 24 * 60 * 60 * 1000 // milliseconds
```
Comment (elv-kevin) [inline src/stores/VideoStore.js:342]: do we really want to make a 100 day token?

**Comment by elv-kevin** [comment]
looks good. we should probably make some sort of embed url generator api utility at some point so we don't have to work with these cryptic small params

File: src/stores/VideoStore.js (hunk 1 above)
Comment (elv-zenia) [reply src/stores/VideoStore.js:342]: Apparently the fabric doesn't honor anything over 24 hours so I guess the value doesn't matter for now

**Comment by elv-arun** [comment]
Verified on demo

**Pull request merged**